### DISCUSSION TITLE
Unable to Edit or Drag Grid Cells 7.0.0-canary.38

### DISCUSSION THREAD

**Discussion by notnull-bmiller** [Comcast/react-data-grid#2582] [Q&A]
**Issue:**
I am probably missing something here, but I have just installed the latest version of react-data-grid on a fresh react app and am unable to edit or drag data in the grid. I have the following modules installed. 

```
    "immutable": "4.0.0-rc.12",
    "react": "17.0.2",
    "react-data-grid": "7.0.0-canary.38",
    "react-dom": "17.0.2",
    "react-scripts": "4.0.3"
```

**Here is an example of the issue:**
https://codesandbox.io/s/boring-haibt-yyvxu?file=/src/components/Example.js

**Example:**
Blue highlight square is present but I am unable to edit or drag.

![image](https://user-images.githubusercontent.com/56919952/113800141-6ee43b00-9724-11eb-87b7-c6146cd0fcf4.png)

**Comment by NoobTW** [comment]
Maybe you need `TextEditor`. Check [this example](https://github.com/adazzle/react-data-grid/blob/85b47f26397573fec86d053334b6059d0f6973c4/stories/demos/AllFeatures.tsx).

**Comment by notnull-bmiller** [comment]
Thank you @NoobTW for responding, but that didn't seem to work in the above example.  In previous releases, there was no specific editor configuration needed.  Was wondering what happened with it working out of the box.  [Example of react-data-grid using version 5.0.5](https://codesandbox.io/s/sparkling-cookies-7on9m?file=/src/components/Example.js)

![image](https://user-images.githubusercontent.com/56919952/117028724-f98c6b80-accb-11eb-9316-874217b25c72.png)
![image](https://user-images.githubusercontent.com/56919952/117028980-38babc80-accc-11eb-9ebf-13dedaddc3db.png)

**Comment by nstepien** [comment]
You must provide an `editor` component on your columns.
